src/pages/joonyoung/Main/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Other from './Ohter';
+import './Sidebar.scss';
+
+const Sidebar = () => {
+  const otherProfileInfos = [
+    { id: 1, username: 'wecode28기', description: 'wecode님외 36명이 follow' },
+    { id: 2, username: 'Younha', description: 'Enna Kim외 10명이 follow' },
+    { id: 3, username: 'wework', description: 'wecode28님외 236명이 follow' },
+  ];
```

Comment:
위와 같이 함수에 영향을 받지 않는 함수나 변수들은 불필요한 메모리할당을 줄이기 위해 함수형 컴포넌트 밖에 선언해주시는 것이 좋습니다. 더불어 위와 같이 변하지 않는 상수 데이터는 아래와 같이 '대문자'와 '_'을 이용해 네이밍하는 것이 컨벤션입니다.
```suggestion
  const OTHER_PROFILE_INFOS = [
    { id: 1, username: 'wecode28기', description: 'wecode님외 36명이 follow' },
    { id: 2, username: 'Younha', description: 'Enna Kim외 10명이 follow' },
    { id: 3, username: 'wework', description: 'wecode28님외 236명이 follow' },
  ];
```